src/pages/Inventory.page.js

```diff
@@ -43,4 +39,37 @@ export class InventoryPage extends BaseSwagLabPage {
     await this.selectSortDropdown.click();
   }
 
+  async getItemInfoById(id,item = this.inventoryItems.nth(id)){
```

Comment:
You need to choose only one way to get necessary element from page:
1 - by id
2 - by provided element
Right now it's not obvious what to use and when